ingestion/setup.py

```diff
@@ -183,7 +182,6 @@ def get_long_description():
     "pytest==7.0.0",
     "pytest-cov",
     "pytest-order",
-    "faker",
```

Comment:
we are not using faker anywhere in the code, removed it..

Reply:
good catch